### PR TITLE
Optimise CVE table cells

### LINKED ISSUE #8832: CVE tables: “Does not exist” wraps to 3 lines
CVE results tables are much less information-dense than they could be, because many of the cells contain the text “Does not exist”, and that text wraps to three lines.

image

(These should also be the least prominent cells in the table, but being almost the _only_ cells that wrap to three lines challenges this.)

These cells have plenty of width for “Does not exist” to occupy two lines rather than three:

image

It doesn’t use this space because it is leaving room for an “ℹ️” icon alongside. However:
- Cells generally don’t need to leave room for this icon; it could float over top on hover.
- There’s no point in “Does not exist” cells particularly leaving room for this icon, because they don’t have it anyway!

What should happen: either
- no cells should leave room for the “ℹ️” icon, because it floats over top if necessary; or
- “Does not exist” cells in particular shouldn’t leave room for the “ℹ️” icon, because they doesn’t have it.

---
*Reported from: https://ubuntu.com/security/cve*

### PR DESCRIPTION
**Pull request by albertkol**
## Done

- Optimise CVE table cells

## QA

Go to /security/cve look at the table it should have:
- `Does not exist` on 2 lines instead of 3
- Update the UA texts under the status, inside the cell and they should be smaller
- It should look better than https://ubuntu.com/security/cve

## Issue / Card

Fixes #8832
Fixes #9036

### CODE REVIEW

File: static/sass/_pattern_cve.scss [hunk 1]
```diff
@@ -71,6 +74,16 @@
     position: relative;
   }
 
+  .cve-table .esm-info {
+    width: 100%;
```
Comment (sowasred2012) [inline static/sass/_pattern_cve.scss:78]: Linting - these should be in alphabetical order

File: static/sass/_pattern_cve.scss [hunk 2]
```diff
@@ -40,11 +40,14 @@
   }
 
   .cve-table .icon-cve-info {
-    margin: 0 0 0 0.5rem;
+    margin: 0;
+    position: absolute !important;
+    top: calc(0.5rem - 1px);
+    right: 0;
```
Comment (sowasred2012) [inline static/sass/_pattern_cve.scss:46]: Same here, alphabetical order

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
Couple of linting issues, and there's a strange 1px border in the gap between the last two columns:
Screenshot from 2021-01-18 12-26-11

File: static/sass/_pattern_cve.scss (hunk 1 above)
Comment (albertkol) [reply static/sass/_pattern_cve.scss]: Fixed!

**Comment by sowasred2012** [comment]
Updates look good, but I just noticed that longer words break out of their cells at smaller browser widths:

Screenshot from 2021-01-19 10-18-05

We could fix this by adding `overflow: hidden; text-overflow: ellipsis;` - unless there are any objections. @matthewpaulthomas ?

Screenshot from 2021-01-19 10-19-12

**Review by sowasred2012 — APPROVED** [review]
LGTM!

**Pull request merged**